packages/fluent/scss/radio/_variables.scss

```diff
@@ -0,0 +1,198 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+@use "../core/functions/" as *; 
+@use "../checkbox/_variables.scss" as *;
+@use "../list/_variables.scss" as *;
+
+// Radio button
+
+/// Border radius of radio button.
+/// @group radio
+$kendo-radio-radius: 50% !default;
+/// Border width of radio button.
+/// @group radio
+$kendo-radio-border-width: 1px !default;
+
+// Radio button sizes
+$kendo-radio-sizes: (
+    sm: (
+        size: map-get( $kendo-spacing, 4 ),
+        glyph-size: ( map-get( $kendo-spacing, 4 ) - map-get( $kendo-spacing, thin ) ),
+        ripple-size: map-get( $kendo-spacing, 4 ) * 3,
+        checked-border-width: map-get( $kendo-spacing, 1 ),
+        checked-position: ( map-get( $kendo-spacing, 1 ) - map-get( $kendo-spacing, hair ))
+    ),
+    md: (
+        size: map-get( $kendo-spacing, 5 ),
+        glyph-size: ( map-get( $kendo-spacing, 5 ) - map-get( $kendo-spacing, thin ) ),
+        ripple-size: map-get( $kendo-spacing, 5 ) * 3,
+        checked-border-width: ( map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, hair )),
+        checked-position: map-get( $kendo-spacing, 1 )
+    ),
+    lg: (
+        size: map-get( $kendo-spacing, 6 ),
+        glyph-size: ( map-get( $kendo-spacing, 6 ) - map-get( $kendo-spacing, thin ) ),
+        ripple-size: map-get( $kendo-spacing, 6 ) * 3,
+        checked-border-width: ( map-get( $kendo-spacing, 1) + map-get( $kendo-spacing, thin )),
+        checked-position: ( map-get( $kendo-spacing, 1) + map-get( $kendo-spacing, hair ))
+    )
+) !default;
+
+/// Default width of radio button.
+/// @group radio
+$kendo-radio-width: map-get( $kendo-spacing, 5) !default;
+/// Default height of radio button.
+/// @group radio
+$kendo-radio-height: map-get( $kendo-spacing, 5) !default;
+/// Default font size of radio button.
+/// @group radio
+$kendo-radio-font-size: ( map-get( $kendo-spacing, 5 ) - map-get( $kendo-spacing, thin ) ) !default;
```

Comment:
can we use --font-size-lg here ?

Reply:
The `--font-size-lg` is 16px but the default font-size is 2 pixels lower than the size of the radio (which is 20px by default) thus the calculation is needed and we cannot use the font-size-lg variable.